aten/src/ATen/autocast_mode.cpp

```diff
@@ -236,16 +236,16 @@ Therefore, for the moment, this is all copy pasted in from VariableTypeEverythin
 // Common cases where registration signature matches redispatch signature
 // (that's why SIGNATURE is repeated in the WrapFunction instantiation)
 #define KERNEL(FUNC, REGISTER_NAME, SIGNATURE, POLICY) \
-  m.impl(REGISTER_NAME, \
+  m.impl(TORCH_SELECTIVE_NAME("aten::" REGISTER_NAME), \
     &WrapFunction<CastPolicy::POLICY, SIGNATURE, SIGNATURE, &FUNC>::type::call);
 
 #define KERNEL_UNBOXED_ONLY(FUNC, REGISTER_NAME, SIGNATURE, POLICY) \
-  m.impl_UNBOXED(REGISTER_NAME, \
+  m.impl_UNBOXED(TORCH_SELECTIVE_NAME("aten::" REGISTER_NAME), \
```

Comment:
impl_UNBOXED with SelectiveStr was implemented but not used. This usage uncovered a bug. It should be 
`return impl_UNBOXED(name.operator const char*(), CppFunction::makeUnboxedOnly(raw_f));`
In library.h. Would you include the update in this PR?